docs/icons.md

```diff
@@ -0,0 +1,37 @@
+# Icons
+
+Geotrek V3 uses icons which images come from API. This file intends to help upload a right format for them in Geotrek Admin.
+
+## Activity icons
+
+Activity icons are used in many places. Their color can be modified depending the context.
+Because of that, they have to be :
+
+- in **`svg` format**
+- **white**, or with a light color, by default
+- **without filled background** : only shapes or lines
+- preferrably, **no different colors**. If different colors are used, they will be lost anyway when the icon is modified. The icon has to stay readable with all its shapes and lines in one color.
+
+**Default activity icons**, not modified, are visible in search filter list or on map :
+![Activity Icon on filter list](assets/iconFilterList.png)
+
+Here that "Alpinisme" icon could have been white by default.
```

Comment:
J'ai peut etre pas compris le sens de ta phrase, mais ça devrait pas être "should" plutot que "could" ? 🤔

Reply:
c'est modifié :)